### PR TITLE
chore(fix): ci - move s3 uploads to only on release

### PR DESCRIPTION
**Pull request by leet4tari**
Description
Moved S3 upload job to be run only on releases.
Also add job to verify all checksums and copy into one file to be gpg signing

Motivation and Context
Remove none release assets from been uploaded to s3, as Windows s3 upload failed often.

How Has This Been Tested?
Run locally and in local fork successfully.

What process can a PR reviewer use to test or verify this change?
Check the binary builds for any errors.

Breaking Changes
With limited access to daily builds in s3, don't believe there will be a visible change.

- [x] None
- [ ] Requires data directory on base node to be deleted
- [ ] Requires hard fork
- [ ] Other - Please specify

### CODE REVIEW

**Review by hansieodendaal — DISMISSED** [review]
utACK

File: .github/workflows/base_node_binaries.yml [hunk 1]
```diff
@@ -314,17 +315,17 @@ jobs:
           TBN_ARCH=$(echo "${${{ matrix.builds.target }}//-apple-darwin/}")
           pkgbuild --root /tmp/tari_testnet \
             --identifier "${{ env.TBN_BUNDLEID_BASE }}.$TBN_BUNDLEID_VALID_NAME" \
```
Comment (hansieodendaal) [inline .github/workflows/base_node_binaries.yml]:
```suggestion
            --identifier "${{ env.TBN_BUNDLED_BASE }}.$TBN_BUNDLED_VALID_NAME" \
```

File: .github/workflows/base_node_binaries.yml [hunk 2]
```diff
@@ -314,17 +315,17 @@ jobs:
...
-            --version "$VERSION" \
+            --version "${TARI_VERSION}" \
             --install-location "/tmp/tari" \
             --scripts "/tmp/tari_testnet/scripts" \
             --sign "Developer ID Installer: $MACOS_INSTALLER_ID" \
-            "${distDirPKG}/${{ env.TBN_FILENAME }}-${{ matrix.builds.name }}-${{ env.VERSION }}.pkg"
+            "${distDirPKG}/${{ env.TBN_FILENAME }}-${{ matrix.builds.name }}-${{ env.TARI_VERSION }}.pkg"
           echo -e "Submitting to Apple...\n\n"
           xcrun altool --notarize-app \
             --primary-bundle-id "${{ env.TBN_BUNDLEID_BASE }}.$TBN_BUNDLEID_VALID_NAME" \
```
Comment (hansieodendaal) [inline .github/workflows/base_node_binaries.yml]:
```suggestion
            --primary-bundle-id "${{ env.TBN_BUNDLED_BASE }}.$TBN_BUNDLED_VALID_NAME" \
```

**Review by hansieodendaal — COMMENTED** [review]
Some typos.

Also 
- `# buid select target images - build_enabled` -> `# build select target images - build_enabled`
- `echo "Checkum verification archive is "` -> `echo "Checksum verification archive is "`

File: .github/workflows/base_node_binaries.yml (hunk 1 above)
Comment (leet4tari) [reply .github/workflows/base_node_binaries.yml]: Thanks, that is meant to be ```BUNDLE_ID_BASE```, let me update

**Review by hansieodendaal — APPROVED** [review]
utACK

**Review by stringhandler — APPROVED** [review]

**Review by SWvheerden — APPROVED** [review]

**Pull request merged**